src/context/schedule.tsx

```
@@ -0,0 +1,48 @@
+  url: string;
+  room: string;
+  [key: string]: any;
+}
+
+export interface ScheduleContextType {
+  schedule: ScheduleItem[];
+  addToSchedule: (
+    dateTime: Date,
+    url: string,
+    room?: string
+  ) => Promise<void | DocumentReference<DocumentData>>;
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)